content/assets/blog/phoenix-case-study/index.md

```diff
@@ -0,0 +1,137 @@
+author: benjamin
+linkedin_link:
+hackernews_link:
+---
+In a world where social media deeply shapes public opinion, data-driven insights can play a pivotal
+role in preventing conflict and fostering peace. That’s exactly what [Build
+Up](https://howtobuildup.org/phoenix/), a nonprofit dedicated to supporting peacebuilders—sought to
+accomplish through Project Phoenix. Developed in partnership with datavaluepeople, Phoenix is an
+open-source (code repository [here](https://gitlab.com/howtobuildup/phoenix)), end-to-end AI
+platform designed to help peacebuilders and mediators analyze social media data, informing
+effective strategies for conflict prevention and resolution. Since January 2025 phoenix has been
+open to the public and you can apply for a Phoenix account,
+[here](https://howtobuildup.org/phoenix/).
+
+In this article, we’ll explore the challenges Build Up faced, the solutions datavaluepeople
+provided, and how Phoenix is transforming social media analysis for peacebuilding around the globe.
+
+***
+
+## **The challenge: Leveraging social media for peacebuilding**
+
+Peacebuilders and mediators increasingly look to social media to gauge public sentiment and
+identify emerging tensions. Yet collecting and analyzing massive volumes of data from diverse
+platforms can be overwhelming. Several roadblocks stood in the way:
+
+1. **Data overload**: Peacebuilders needed a reliable way to gather, organize, and filter vast
+   streams of social media data in real-time.
+
+2. **Limited technical expertise**: Many peacebuilders aren’t data scientists, so any AI-driven
+   platform had to be intuitive and accessible.
+
+3. **Multi-language support**: Conflicts often span regions with multiple languages, including
+   those that traditional analytics tools rarely handle well.
+
+4. **Open-source requirement**: Build Up wanted a project open to contributors worldwide,
+   facilitating collaboration and continuous improvement.
+
+***
+
+## **The solution: Phoenix—An AI platform for peacebuilders**
+
+Working closely with Build Up, datavaluepeople created Phoenix: a powerful, user-friendly system
+that addresses each challenge in a single, comprehensive platform.
+
+### **Modern web console**
+
+Built with React and Next.js, Phoenix’s web console offers a straightforward interface accessible
+from any device with an internet connection. It allows peacebuilders to easily configure analyses,
+explore data sources, and review insights without advanced technical skills.
```

Comment:
I think this is leaving out reference to "scraping" or "data collection" which is an integral part.

Reply:
f4987d5